voc/python/ast.py

```diff
@@ -880,6 +896,12 @@ def visit_Break(self, node):
             if loop.end_op is None:
                 break
         self.context.add_opcodes(
+            java.New('org/python/types/Bool'),
+            JavaOpcodes.ICONST_0(),
+            java.Init('org/python/types/Bool', 'Z'),
+            ASTORE_name('#for-orelse-%x' % id(loop))
+        )
```

Comment:
I don't doubt that this works, but it's also going to be a redundant operation anywhere that you use a break that *isn't* part of a For, isn't it? For example, if you break from a while, it certainly won't *hurt* to have this extra bytecode, but it would be nice if it didn't have to run.